src/modules/Cluster.rb

```diff
@@ -298,6 +330,24 @@ def generateMemberString(memberaddr)
       return address_string
     end
 
+    def generateDictString(obj)
```

Comment:
Maybe you could `require 'json'` and use `obj.to_json` instead.

Reply:
Thanks I will try this.

Reply:
Seems `json` format is incompatible with SCR.Write(). The later one will automatically add `"` to the strings, then SCR.Read() will fail because `obj.to_json` also define property name enclosed in `"`. For example, string `{"exec_ping":"ping -q -c 1 \"127.0.0.1\""}` will change to `"{"exec_ping":"ping -q -c 1 \"127.0.0.1\""}"` in SCR.write(), then get the string like `"{"` instead of the hash.